ui/src/core_components/content/CoreLink.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+	<div class="CoreLink">
```

Comment:
Since you're working on links, can you please check how the Markdown links are working, if everything is alright with them?

I assume that with a _Text component_, markdown mode, I can `[hello](https://streamsync.cloud)`.

Reply:
Just checked and yes - you can. You can also `[Next page](#next-page)` to navigate between pages.

Reply:
So it's basically the same, only difference is that you can select page in component and define rel and target.

Reply:
Thanks for checking!

Yes they're quite similar, that's what kept me from creating this component. However, now that I think more about it:
- Agreed, there's the `rel` and `target` 
- You can change the color
- It's more semantic. If you see a component tree with 3 _Link_, you it's self-explanatory. You see 3 _Text_, could be anything.
- It's more easily discoverable. "I need to create a link, so I'll create a _Text_ component which probably supports markdown and I'll `[]()` my way into it." -Probably no one ever, at least not on their first try.

By the way there's the possibility of opening URLs from the backend via backend actions too.